### PR TITLE
Multiple workflows

### PR DESCRIPTION
**Pull request by rstyd**
This PR addresses #271. There were a ton of conflicts so currently making sure everything works as it should.

### CODE REVIEW

**Comment by pagrubel** [comment]
Since this is  a large PR I'd like to see 2 reviews before merging, so I'm assigning Jake and myself.

**Comment by pagrubel** [comment]
Deleted that last comment, but I do think we should have 2 approvals before merging this.

**Comment by jtronge** [comment]
Some of the unit tests are failing because of changes to the gdb start/stop code. I wonder if there's some way we can make it easier to start the gdb for testing.

**Comment by jtronge** [comment]
I get this when trying to run the cat-grep-tar workflow: `AttributeError: module 'beeflow.wf_manager.resources.wf_utils' has no attribute 'create_workflow_dir'`

**Comment by jtronge** [comment]
Ok, I tried submitting two clamr workflows at the same time to different workdirs and they both worked.

Do you think we could add the workflow id (or the short version) in the log output. Something like this maybe:

```
INFO: Submitted ['clamr'] to Task Manager (wf_id: a55624)
INFO: Workflow Completed (wf_id: a55624)
```

**Comment by pagrubel** [comment]
Show me that stderr is working I cannot get output from ffmpeg other than our BEE output in the .beeflow/workflows dir

File: beeflow/tests/gdb.py
Comment (pagrubel) [inline beeflow/tests/gdb.py:9]: Why are you keeping this around?

File: beeflow/tests/gdb.py
Comment (pagrubel) [inline beeflow/tests/gdb.py:10]: Is this dangerous to keep this around?

**Comment by pagrubel** [comment]
> Show me that stderr is working I cannot get output from ffmpeg other than our BEE output in the .beeflow/workflows dir

Okay we decided that we should make sure stderr is fully supported for the release but it should be fixed in another PR

**Review by jtronge — APPROVED** [review]

**Review by pagrubel — APPROVED** [review]

**Pull request merged**
